lib/dmv_factory.rb

```diff
@@ -0,0 +1,24 @@
+class DmvFactory 
+  attr_reader :office_list
+  def initialize
+    @office = nil
+  end
```

Comment:
The instance variable `@office` is not being used anywhere in the code and could probably be removed.